packages/trimerge-sync-indexed-db/src/IndexedDbCommitRepository.ts

```diff
@@ -78,7 +86,7 @@ export async function resetDocRemoteSyncData(docId: string): Promise<void> {
 export function getIDBPDatabase<CommitMetadata, Delta>(
   docId: string,
 ): Promise<IDBPDatabase<TrimergeSyncDbSchema<CommitMetadata, Delta>>> {
-  return createIndexedDb(getDatabaseName(docId));
+  return getDocDatabase(getDatabaseName(docId));
```

Comment:
```suggestion
  return getDocDatabase(docId);
```

Reply:
hmmm we probably don't need to export both of these